query-engine/core/src/schema_builder/input_types/input_fields.rs

```diff
@@ -279,7 +279,10 @@ where
                 object_fields.push(
                     input_field(
                         operations::PUSH,
-                        map_scalar_input_type(ctx, &f.type_identifier, false),
+                        vec![
+                            map_scalar_input_type(ctx, &f.type_identifier, false),
+                            list_input_type.clone(),
+                        ],
                         None,
                     )
```

Comment:
Does the order of the types of a field matter? I'm asking to know whether we should have the list type first or whether it doesn't matter.

Reply:
It doesn't matter here, no.